packages/url-loader/src/lib/upload.ts

```diff
@@ -0,0 +1,40 @@
+import { CloudinaryUploadWidgetOptions } from '@cloudinary-util/types';
+
+/**
+ * generateSignature
+ * @description Makes a request to an endpoint to sign Cloudinary parameters as part of widget creation
+ */
+
+export interface GenerateSignatureCallback {
+  fetch: Function;
```

Comment:
might be able to use `typeof fetch` here which should be globally available

Reply:
good point, though my original thinking was someone could pass in something that's _not_ fetch. realistically its going to be fetch and the shape supports fetch "as is", so perhaps i just move forward with the suggestion and can look at other use cases should they ever come up